src/Data/Nat/Properties.agda

```diff
@@ -225,6 +225,9 @@ n≤1+n _ = ≤-step ≤-refl
 n≤0⇒n≡0 : ∀ {n} → n ≤ 0 → n ≡ 0
 n≤0⇒n≡0 z≤n = refl
 
+1+n≤1+m⇒n≤m : ∀ {m n} → suc m ≤ suc n → m ≤ n
+1+n≤1+m⇒n≤m (s≤s le) = le
+
```

Comment:
Is this [`pred-mono`](http://agda.github.io/agda-stdlib/Data.Nat.Properties.html#11633)?

Reply:
I expected its name to be `suc-inj` or `suc-injective` !!

Reply:
> I expected its name to be suc-inj or suc-injective !!

Err, but we already use that name for injectivity over equality.

Reply:
Oh shoot, my bad - I somehow bleeped over the `≤`.  Sorry for the noise.

Reply:
@gallais Wouldn't `pred-mono` indicate the property `m ≤ n → pred m ≤ pred n`? Which is similar but slightly different property?